### DISCUSSION TITLE
Will you be switching to yugabyte now that the cockroachdb license has changed?

### DISCUSSION THREAD

**Discussion by lknite** [zitadel/zitadel#8950] [General]
Yugabyte seems the obvious next choice, very similar to cockroachdb while also 100% open source.

**Comment by fforootd** [comment]
While Yugabyte has been on our radar for a while it was so far not an actual option since they only supported PG version 11. 
Recently they started to work on PG 15 support which we had some discussion around here https://github.com/zitadel/zitadel/discussions/8840

So our general stand is that we optimize plain PG >= 15 since this is ubiquitously available in IaaS providers and can be run locally with a low resource footprint. For performance related topics we just recently released btw. a cache option 😁 

When Yugabyte PG15 support drops we certainly will look into it again but we do not have a clear timeline for adopting Yugabyte since no decision can be made at the moment.

**Comment by skillpoint-dev** [comment]
I tried running Zitadel with YugabyteDB 2.25.1 and it seems to be working for now - migration of Postgres Schema and Data using Yugabyte Voyager showed no problem with schema or data - so I think at least tests with this combination can be started if anybody was wondering.
